test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -443,6 +443,20 @@ static int test_overall_capabilities(void)
 				CU_ASSERT(per_level->max_priority != 0);
 				return -1;
 			}
+
+			if (per_level->tm_node_shaper_supported) {
+				CU_ASSERT(per_level->max_burst > 0);
+				CU_ASSERT(per_level->min_burst > 0);
```

Comment:
I put a comment in the wrong place, but this is where I wonder why burst limit of zero is not ok.